compiler/circle-interpreter-cffi-test/README.md

```diff
@@ -1,11 +1,15 @@
 # circle-interpreter-cffi-test
 
-The `circle_interpereter_cffi` library wrapped with CFFI is designed to expose an existing `luci-interpreter` class to Python. It simplifies the integration of the class by creating a Python-compatible interface using CFFI.
+<br>The `circle_interpereter_cffi` library wrapped with CFFI is designed to expose
```

Comment:
May I ask why use `<br>...</br>` ?

Reply:
I used this to render the text one line with reducing the number of column. Is there any good idea to do this?

Reply:
Plz try like
```
This is some
line that should render
in one line.

This is another
line that should render in another line.
```
which should render like
```
This is some line that should render in one line.
This is another line that should render in another line.
```

Reply:
It works! I tried that just a minute ago but I did something wrong at that time. Thanks:)